cmd/juju/cloud/list.go

```diff
@@ -28,17 +28,27 @@ type listCloudsCommand struct {
 // listCloudsDoc is multi-line since we need to use ` to denote
 // commands for ease in markdown.
 var listCloudsDoc = "" +
-	"Provided information includes 'cloud' (as understood by Juju), cloud\n" +
-	"'type', and cloud 'regions'.\n" +
-	"The listing will consist of public clouds and any custom clouds made\n" +
-	"available through the `juju add-cloud` command. The former can be updated\n" +
-	"via the `juju update-clouds` command.\n" +
-	"By default, the tabular format is used.\n" + listCloudsDocExamples
+        "Output includes fundamental properties for each cloud known to the\n" +
+        "current Juju client. These are: name, number of regions, default region,\n" +
+	"type, and description.\n" +
+        "\nThe cloud name is what's used to refer to a cloud in any Juju command.\n" +
+        "\nThe default output shows public clouds known to Juju out of the box\n" +
+	"(these may change between Juju versions). In addition to these public\n" +
+	"clouds, the 'localhost' cloud (local LXD) is also listed.\n" +
+        "\nThis command's default output format is 'tabular'.\n" +
+        "\nCloud metadata sometimes changes, e.g. AWS adds a new region. Use the\n" +
+	"`update-clouds` command to update the current Juju client accordingly.\n" +
+	"\nUse the `add-cloud` command to add a private cloud to the list of clouds\n" +
+	"known to the current Juju client.\n" +
+        "\nUse the `regions` command to list a cloud's regions and the `show-cloud`\n" +
```

Comment:
Let's not compound these 2 sentences. I prefer a separate sentence for each command.